onthequad/onthequad/UserEndPoints.py

```diff
@@ -0,0 +1,79 @@
+import falcon
+import json
+from .UsersDB import UsersDB
+from .DataBaseSetUp import DataBaseSetUp
+authen = DataBaseSetUp.authentication()
+
+class UserEndPoints:
+
+    def on_get(self, req, resp):
+        
+        send = UsersDB.getAllUsers()
+        resp.body = json.dumps(send)
+        resp.status = falcon.HTTP_200
+
+    def on_post(self, req, resp):
+        data = json.loads(req.stream.read())
+    
+        if any(data):
+
+            user = authen.create_user_with_email_and_password(data['email'], data['password'])
+            uId = user['localId']
+            del data['password']
+            emailArray = data['email'].split("@")
+            userName = emailArray[0]
+            data['userId'] = uId
```

Comment:
Everything looks great, but I think we can live without the UID inside the entry since it lives on top now.

Reply:
The dummy json has userId in the body. I want to left it out too but I think it would be better to following the format of the dummy json since it does not hurt to left it there.